### PR TITLE
[Feature] Add settings option for json net serializer

### PR DESCRIPTION
**Pull request by jonrad**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/jonnii/chinchilla/pull/71" target="_blank">ReviewNinja

File: src/Chinchilla.Serializers.JsonNET/JsonNetMessageSerializer.cs [hunk 1]
```diff
@@ -5,6 +5,17 @@ namespace Chinchilla.Serializers.JsonNET
 {
     public class JsonNetMessageSerializer : IMessageSerializer
     {
+        private readonly JsonSerializerSettings serializerSettings;
```
Comment (jonnii) [inline src/Chinchilla.Serializers.JsonNET/JsonNetMessageSerializer.cs]: it's ok if this is left null?

Comment (jonrad) [inline src/Chinchilla.Serializers.JsonNET/JsonNetMessageSerializer.cs]: @jonnii Yes, that's why the rest of the tests still work

**Comment by jonnii** [comment]
:+1:

**Pull request merged**